src/services/users.services.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const { generateId } = require('../helpers/helpers.js');
+
+/**
+ *@typedef {Object} User
+ *@property {number} id
+ *@property {string} name
+ *@type {User[] | []}
+ */
+let users = [];
+
+const getAll = () => {
+  return users.length !== 0 ? users : [];
```

Comment:
```suggestion
  return users;
```
It doesn't make much sense